sharktank/sharktank/ops/__init__.py

```diff
@@ -18,7 +18,75 @@
 
 from . import _registry
 from ..types.tensors import unbox_tensor
-from .signatures import *
+
+def import_and_wrap_signatures():
```

Comment:
This ended up having to go in the \_\_init\_\_ file so that any import of ops would wrap the functions.

One problem with the approach: a subsequent run of `from .signatures import *` will override the wrapped versions.

Reply:
I am pretty certain we don't want to do this. Only the `sharded_impls` should be the one to consider transfer information. This is pretty specific to sharded impls. We only want to wrap the importers for the sharded cases.

Reply:
Also, try to name this `import_sharded_signatures` or something to that effect.